### ISSUE TITLE
Build fails due to change to `show_image` function in darknet

### ISSUE THREAD

**Issue by halhenke** [madhawav/YOLO3-4-Py#28]
This change was made 15 days ago:

https://github.com/pjreddie/darknet/commit/b13f67bfdd87434e141af532cdb5dc1b8369aa3b#diff-06d7eaa529ff0d43e382b4b216f391c0L708

which leads to:

```
Step #0 - "yolo-3-4": pydarknet.cpp: In function 'PyObject* __pyx_pf_9pydarknet_5Image_2show_image(__pyx_obj_9pydarknet_Image*, char*)':
Step #0 - "yolo-3-4": pydarknet.cpp:2233:46: error: too few arguments to function 'int show_image(image, const char*, int)'
Step #0 - "yolo-3-4": show_image(__pyx_v_self->img, __pyx_v_title);
Step #0 - "yolo-3-4": ^
Step #0 - "yolo-3-4": In file included from bridge.h:9:0,
Step #0 - "yolo-3-4": from pydarknet.cpp:587:
Step #0 - "yolo-3-4": /darknet/include/darknet.h:713:5: note: declared here
Step #0 - "yolo-3-4": int show_image(image p, const char *name, int ms);
Step #0 - "yolo-3-4": ^
Step #0 - "yolo-3-4": error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
Step #0 - "yolo-3-4": The command '/bin/sh -c python3 setup.py build_ext --inplace' returned a non-zero code: 1
Finished Step #0 - "yolo-3-4"
ERROR
ERROR: build step 0 "gcr.io/cloud-builders/docker" failed: exit status 1
Step #0 - "yolo-3-4":
```

**Comment by madhawav** [comment]
Thanks, @halhenke for showing the issue. Will look into this and release a fix soon.

**Comment by madhawav** [comment]
@halhenke For the time being, you can use the fork of darknet I have made. The code available in it is compatible with yolo34py.
Fork Link: https://github.com/madhawav/darknet
Also, choose the appropriate branch from the above repo.

**Comment by madhawav** [comment]
Fixed with #30 merged to master.
